components/camera/server.go

```diff
@@ -53,8 +57,28 @@ func (s *subtypeServer) GetImage(
 		return nil, err
 	}
 
+	// Determine the mimeType we should try to use based on camera properties
 	if req.MimeType == "" {
-		req.MimeType = utils.MimeTypeRawRGBALazy
+		if s.imgType == nil {
+			props, err := cam.Properties(ctx)
+			if err != nil {
+				s.logger.Warn("camera properties not found, assuming color images")
```

Comment:
Put in the camera name as well - 
`Warnf("camera properties not found for %q, assuming color images", req.Name)`

Reply:
Also, wrap the error message that you did get into the warning

Reply:
Done